tests/test_dstft.py

```
@@ -159,6 +159,18 @@ def test_forward_before_initialize_raises() -> None:
 # ---------------------------------------------------------------------------
 
 
+def test_initialize_rejects_non_tensor_input() -> None:
+    dstft = DSTFT(n_fft=64)
+    with pytest.raises(TypeError, match="must be a torch.Tensor"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)